code/src/components/TaskList.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import tasks from 'reducers/task';
+
+const TaskList = () => {
+  const taskList = useSelector((backpack) => backpack.tasks.items);
+
+  const dispatch = useDispatch();
+
+  const onIsDoneToggle = (id) => {
+    dispatch(tasks.actions.toggleItem(id));
+  }
+  const onDeleteTaskBtnClick = (taskIndex) => {
+    dispatch(tasks.actions.deleteItem(taskIndex));
+  }
+  return (
+    <section>
+      {taskList.map((singleTask, index) => {
+        return (
+          <article className="taskListBox" key={singleTask.id}>
+            <h2>{singleTask.name}</h2>
+            <label className="Is this task done">
+              <input type="radio" checked={singleTask.isDone} onChange={() => onIsDoneToggle(singleTask.id)} />
```

Comment:
I tried to find why you can't uncheck your boxes after checking them. Maybe you need to put a id={singleTask.id} in the input on line 24, or maybe it has something to do with it being a radiobutton and not a checkbox. 🤷‍♀️ I also struggled a lot with checkbox-errors so I'm probably not the right person to try to find the solution. 😅

Reply:
Hey, Linnéa! Thank you for commenting :D It's because I have the radio and not the checkbox button. Found it out.. Tried with radio at some point and didn't turn it back to checkbox. Yeah does boxes were tricky :( Typical! Thank you a lot for your input!!